src/mumble/UserView.h

```diff
@@ -17,14 +17,14 @@ class UserDelegate : public QStyledItemDelegate {
 	Q_OBJECT
 	Q_DISABLE_COPY(UserDelegate)
 
-	int m_flagTotalDimension;
-	int m_flagIconPadding;
-	int m_flagIconDimension;
-
 public:
-	UserDelegate(QObject *parent, int flagTotalDimension, int flagIconPadding, int flagIconDimension);
+	UserDelegate(QObject *parent);
 	void paint(QPainter *painter, const QStyleOptionViewItem &option, const QModelIndex &index) const Q_DECL_OVERRIDE;
 
+	const static int FLAG_ICON_PADDING = 1;
+	static int FLAG_TOTAL_DIMENSION;
+	static int FLAG_ICON_DIMENSION;
```

Comment:
Those shouldn't be static, as they are not used as static variables. Instead each instance of a `UserDelegate` will set these values repeatedly and the underlying assumption is that they always set it to the same value. However, I would consider it to be cleaner if these were member variables.

Reply:
These are static, because the Qt method ``paint`` is const and does not allow to change members.

Reply:
In this case it would still be better to make them members and mark them as `mutable`. But the even better solution would be, if we can get the constructor-based processing to work.

Reply:
Ok fair. But this would require me to save a pointer to UserDelegate in UserView to be able to access the value later. This could be done, but we would an additional commit for 1.5.0 for using a qt_unique_ptr ;)